opentelemetry-otlp/Cargo.toml

```diff
@@ -36,7 +36,8 @@ futures-util = { version = "0.3", default-features = false, features = ["std"] }
 opentelemetry-proto = { version = "0.2", path = "../opentelemetry-proto", default-features = false }
 
 grpcio = { version = "0.12", optional = true}
-opentelemetry = { version = "0.18", default-features = false, features = ["trace"], path = "../opentelemetry" }
+opentelemetry_api = { version = "0.18", default-features = false, path = "../opentelemetry-api" }
```

Comment:
Not sure why in dynatrace you chose to go with the version without the path, but chose to keep the path in this crate.

Reply:
dynatrace hasn't yet been updated, can do that in a follow up